elliottlib/cli/find_bugs_wo_target_version.py

```diff
@@ -0,0 +1,54 @@
+import json
+import click
+from elliottlib import (Runtime, logutil)
+from elliottlib.cli import common
+from elliottlib.cli.find_bugs_sweep_cli import FindBugsSweep, print_report
+
+logger = logutil.getLogger(__name__)
+
+
+@common.cli.command("find-bugs:notv", short_help="Find qualified ART managed jira bugs (sweep) that do not have a "
```

Comment:
Maybe we can make this command more generic, like `find-bugs:invalid`, to detect multiple problems for issues inside OCPBUGS project. e.g.
- Bugs without valid target version
- Reopened bugs
- CVE trackers without required labels or CVE name in the title

We don't have to implement the other 2 checks for now, but it would be nice if we can implement a schedule job to to find invalid bugs in the future to report all problematic bugs. Thoughts?

Reply:
Really like the idea yuxiang, this can be a catch all for all those invalid cases, and we can fit it in a job. I'll update the PR - rename the command and add some way to specify the checks

Reply:
Coming back to this. I took some time to look at verify-bugs command and there is some overlap with this. It already does some of the trackers without labels / cve name in the title validation.
Filed https://github.com/openshift-eng/elliott/pull/529 to update the help

Target release is really a special case, instead of shoe-horning this and verify-bugs, I really like @joepvd 's idea - https://github.com/openshift-eng/elliott/pull/522#pullrequestreview-1328405875. So we separate out named queries in yml and then keep a generic search  to search and then another to comment. I'll write up a story for that